Tasks/Anastasia Konuyk/Task1/task1.py

```diff
@@ -0,0 +1,10 @@
+deposit = 20000
+percent = 15/100
+years = 5
```

Comment:
these values should of the decimal data type, since they are money-related, -1 point for the unwise choice